tests/feature/Rules/LengthTest.php

```diff
@@ -41,3 +41,18 @@
     '- The length of Cactus must be equal to 3',
     ['lengthEquals' => 'The length of Cactus must be equal to 3']
 ));
+
+test('Chained wrapped rule', expectAll(
+    fn() => v::length(v::between(5, 7)->odd())->assert([]),
```

Comment:
This is pretty much the essence of the changes. 

If you see the other tests, `length()` receives only 1 rule, this test receives two rules, and the two of them fail. That means that we can't simply add the prefix ("The length of") to the result of that rule because the result has two children.

When `Length` receives a rule that produces a result with children, we need to add the prefix ("The length of") to all the children of that result.

Reply:
@dmjohnsson23 I wrote some comments to this merge request. If you have any questions, you can just write here, I'll be happy to answer them!